util/dashboard/gen_dashboard_entry.py

```diff
@@ -33,11 +35,103 @@ def genout(outfile, msg):
     'V3': 'Verification Complete'
 }
 
+# TODO: This is relative to the dashboard, which is currently located at
+# hw/_index.md.
+docs_server = "../.."
+
 
 def convert_stage(stagestr):
     return STAGE_STRINGS.get(stagestr, "UNKNOWN")
 
 
+# Link name with design spec
+def get_linked_design_spec(obj):
+    result = ""
+    if 'design_spec' in obj.keys():
+        design_spec_url = docs_server + "/" + html.escape(obj['design_spec'])
+        result = "<a href=\"{}\">{}</a>".format(
+            design_spec_url, html.escape(obj['name'].upper()))
+    else:
+        result = html.escape(obj['name'].upper())
+
+    return result
+
+
+# Link name with dv_plan
+def get_linked_dv_plan(obj):
+    if 'dv_plan' in obj.keys():
+        dv_plan_url = docs_server + "/" + html.escape(obj['dv_plan'])
+        return "<a href=\"{}\">{}</a>".format(dv_plan_url, "DV Plan")
```

Comment:
I'm still trying to slim down, and this "two line" `DV plan` looks
clunky. Maybe just put "DV" here within the cell text so that it
doesn't get broken? It still has "DV plan" at the cell header. wdyt?

Reply:
Agree, this does look better. I updated the table (and the mockup). Thanks!